db/queries.py

```diff
@@ -3,44 +3,74 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(title__contains="Untitled")
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(User.objects.filter(
+        message__sent__year=2015
+    ).values_list("first_name", "last_name"))
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(
+        message__sent__year__gte=2020
+    ).distinct())
 
 
 def get_messages_contain_authors_first_name() -> list[Message]:
-    pass
+    return list(Message.objects.filter(
```

Comment:
I'm pretty sure this statement is shorter than 79 symbols. You don't really have to split it.